src/components/generics/ConstantBasedPicker.js

```diff
@@ -45,7 +45,7 @@ class ConstantBasedPicker extends Component {
       readOnly = false,
       required = false,
     } = this.props;
-    const { value } = this.state;
+    const { value } = this.state?.value === undefined? {value: null}: this.state;
```

Comment:
This looks quite complicated, is 
```suggestion
    const value = !!this.state ? this.state.value : null
```
will it serve the same function?

Reply:
Unfortunately provided suggestion does not work.

Reply:
But I found more elegant way.